### ISSUE TITLE
High loss and CUDA version

### ISSUE THREAD

**Issue by dasmehdix** [pjreddie/darknet#1748]
Hi guys,
I am trying to train Yolo V3 with my custom dataset.Dataset, xml format, cfg files they all done.I set batch size and subdivision 64 and set filters.

The current avg. loss is =1800.95 

It is too much,How can i decrease it?
When i use tensorflow ssd model the loss start from 5 and converges to 0,2.

So i check the cuda and nvidia see cuda as 10.1 version but other side cuda looks 10.0.I only installed 10.0 version.
Please help me

![cuda](https://user-images.githubusercontent.com/52709620/62112574-a21dc800-b2bb-11e9-80cf-1fba1125aa22.png)
